spec/factories/disk.rb

```diff
@@ -1,4 +1,11 @@
 FactoryGirl.define do
   factory :disk do
+    trait(:hdd) do
+      device_name "test_disk"
```

Comment:
you're right, I left them there as the 1st plan was to have full data about HDD factory, but for my test's purpose I don't need them, I'm going to remove the unnecessary attributes in updated PR.